recipes/libtorrent/all/conanfile.py

```diff
@@ -88,13 +89,14 @@ def _configure_cmake(self):
         self._cmake.definitions["build_tools"] = False
         self._cmake.definitions["python-bindings"] = False
         self._cmake.definitions["python-bindings"] = False
-        self._cmake.configure(build_folder=self._build_subfolder)
+        self._cmake.configure()
         return self._cmake
 
     def _patch_sources(self):
-        for patch_data in self.conan_data["patches"][self.version]:
+        for patch_data in self.conan_data.get("patches", {}).get(self.version, []):
             tools.patch(**patch_data)
 
+        tools.replace_in_file(os.path.join(self._source_subfolder, "CMakeLists.txt"), "/W4", "")
```

Comment:
Why do we need to disable warnings?

Reply:
I think it is done for convenience (`/W4` spews **a lot** of warnings).

Reply:
that's it